_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_articles.scss

```diff
@@ -69,16 +203,106 @@
       text-align: left;
       padding: var(--pf-global--spacer--sm) 0 var(--pf-global--spacer--sm) var(--pf-global--spacer--sm);
     }
+     p:first-child:after {
+        content: '\201d';
+        color: #bb301b;
+        font-size: 3em;
+        font-weight: 700;
+        position: absolute;
+        bottom: 0;
+        right: 2.5rem;
+        text-shadow: none;
+        z-index: -300;
+    }
   }
-  blockquote p:first-child:after {
-      content: '\201d';
-      color: #bb301b;
-      font-size: 3em;
+}
+
+
+.page-node-type-coding-resource {
+  .gsi-meta {
+    margin-bottom: 24px;
+    overflow: hidden;
+    background: #F9F9F9;
+    padding: 12px;
+    border: 1px solid #E6E6E6;
+    border-radius: 0;
+
+    .field--name-field-related-product {
+      &.field--type-entity-reference {
+        background: none;
+        border: none;
+        padding: 0;
+        margin: 0;
+      }
+    }
+
+    .gsi-nav {
+      background-color: #f9f9f9;
+      border-top: 2px solid #0066CC;
+      font-size: 16px;
       font-weight: 700;
+      line-height: 24px;
+      padding: 12px 16px;
+      transform: translateY(0);
+      transition: none;
+      list-style: none;
+      margin: 0;
+
+      li {
+        margin: 0;
+        padding: 4px 0;
+
+        a:hover {
+          color: #004c98;
+          text-decoration: none;
+        }
+
+        &:first-of-type {
+          padding-top: 0;
+        }
+
+        &:last-of-type {
+          padding-bottom: 0;
+        }
+      }
+    }
+
+    ul {
+      margin-top: 0;
+      padding-left: 15px;
+
+      li {
+        width: 33.33%;
+        float: left;
+        list-style: none;
+
+        strong {
+          margin-right: 3px;
+        }
+
+        a+a {
+          margin-left: 5px;
+        }
+      }
+    }
+  }
+  .assembly-type-content_pull_quote {
```

Comment:
The pull quote on the page at /testpage/ does not match what's in production.  It's the only example of that assembly, but it's not clear why content team has not used this assembly.